Astro release notes - August 10 (#1018)
astronomer/docs: astro/release-notes.md

**Latest CLI Version**: 1.3.0 ([Release notes](cli/release-notes.md))

## August 10, 2022

> ### Updated Organization and Workspace role permissions

Reviewer: ```suggestion
### Updated user permissions for Organization and Workspace roles
```

> Some user roles have received new and modified permissions:

Reviewer 2: ```suggestion
The following user roles have new and modified permissions:
```

> - Organization Owners now have Workspace Admin permissions for all Workspaces in their Organization. This role now provides complete access to Workspaces, Deployments, and usage across your Organization.

Reviewer 2: ```suggestion
- Organization Owners now have Workspace Admin permissions for all Workspaces in their Organization. This role can now access Organization Workspaces, Deployments, and usage data.
```

- Organization Billing Admins can now view [usage](deployment-metrics.md#astro-usage) for all Workspaces in their Organization regardless of their Workspace permissions.

> - Workspace Editors can now delete any Deployments in their Workspace.

Reviewer: ```suggestion
- Workspace Editors can now delete any Deployment in their Workspace.
```

> ### Automatic role assignment for new users authenticating with an identity provider

Reviewer: ```suggestion
### Automatic access for new users authenticating with an identity provider
```

If your organization has [implemented an identity provider (IdP)](configure-idp.md), any new user who authenticates to Astro through your IdP is now automatically assigned the Organization Member role. This means that users authenticating through your IdP do not need to be invited by email before joining your Organization.

### Additional improvements

> - Added guardrails so that you can have a Workspace role only if you have an Organization role in the Organization where the Workspace is located.

Reviewer: ```suggestion
- Added a security measure that ensures Workspace roles can only be assigned to users who have an Organization role in the Organization in which the Workspace is hosted. This ensures that a user who does not belong to your Organization cannot be assigned a Workspace role within it.
```
